### ISSUE TITLE
Styles missing when accessing app through ip-address

### ISSUE THREAD

**Issue by 0nn0** [gaearon/react-hot-loader#121]
** QUESTION **

Hi there,

React-hot-loader is amazing and has significantly improved my workflow. One point I haven't figured out yet regards testing on other devices. 

When you have your app running on 0.0.0.0:[port] you can access it on other devices that are on the same network using your ip address. The reason I'd like to do this is so that I can run quick tests on other devices but also on virtual machines with IE.

The problem is that the styles that get injected in the page using the style-loader are not available when accessing the app as described above. What is the best way to solve this? Any tips?

I'm thinking of resorting to using the ExtractTextPlugin to output the CSS in a file, and using a Gulp task to reload the page. However, I was wondering if there's a more elegant way to handle this using react-hot-loader.

**Comment by gaearon** [comment]
>The problem is that the styles that get injected in the page using the style-loader are not available when accessing the app as described above. What is the best way to solve this? Any tips?

Why aren't they? CORS problems?

**Comment by 0nn0** [comment]
I tried it with the react-hot-boilerplate setup, where I added a css file with some styles and changed "localhost" in server.js to "0.0.0.0". There it works like a charm and the styles. 

The problem does arise once I use a setup like isomorphic-hot-loader where you run an express server next to the webpack-dev-server. Somehow the styles-tag does not get injected into the header.

I'll close the topic since the problem doesn't seem to be related to react-hot-loader.
However, any tips on the problem I'm describing above would be much appreciated.
